Core/clim-core/panes/stream-panes.lisp

```diff
@@ -365,49 +365,50 @@ current background message was set."))
 ;;;
 ;;; CONSTRUCTORS
 ;;;
-
 (defun make-clim-stream-pane (&rest options
                               &key (type 'clim-stream-pane)
+                                label
+                                label-alignment
+                                (display-after-commands nil display-after-commands-p)
```

Comment:
Should an error be signaled if both `display-after-commands` and `display-time` are supplied?

Reply:
I have added the error